### DISCUSSION TITLE
How do I set the window title and icon

### DISCUSSION THREAD

**Discussion by erikgassler** [slint-ui/slint#4281] [General] [answered]
I have a new Slint desktop application that I am building using rust and when I run it either directly from `cargo run` or through the built exe executable the window that opens has a generic Windows icon and `Slint Window` in the title bar.
How can I update my app to use a custom icon and set the title to what I want?

**Comment by tronical** [comment, marked as answer]
You can set the title and icon using the title and icon properties, like here:

https://github.com/slint-ui/slint/blob/fd6d51528f097202e4e43168a9b1baff271af43c/examples/gallery/gallery.slint#L14

**Reply by erikgassler** [reply]
Thank you. I think it would be helpful to add this type of information to the default templates - i.e. `rust-slint-template` - as well as to the starter tutorials.

**Reply by tronical** [reply]
Good idea :)
